latex-syntax/src/main/java/org/xwiki/contrib/latex/internal/TemplateProcessor.java

```diff
@@ -98,18 +114,8 @@ public String process(Collection<Block> blocks)
         return writer.toString();
     }
 
-    /**
-     * @param block the block for which to find a template
-     * @return the template for the passed block
-     * @throws Exception if the template cannot be found
-     */
-    public Template getTemplate(Block block) throws Exception
+    private Template getTemplateByName(String templateName) throws Exception
```

Comment:
I don't understand the need for `getTemplateByName()`. Why would the caller not  call `getTemplate()` directly?

Reply:
Indeed, I'd missed that it became useless at some point in my refactoring. I'll remove it. Thanks.